edgexfoundry/edgex-examples
# refactor: Upgrade cloud-export-mqtt to use V3

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:


## Testing Instructions
Start edgex with no-secty
run `./app-service -p aws -cp -o -dev`
verify service starts up
run `./app-service -p azure -cp -o -dev`
verify service starts up
